docs/maven.rst

```diff
@@ -0,0 +1,51 @@
+Using the SpotBugs Maven Plugin
+===============================
+
+This chapter describes how to integrate SpotBugs into a Maven project.
+
+Add spotbugs-maven-plugin to your pom.xml
+-----------------------------------------
+
+Add ``<plugin>`` into your ``pom.xml`` like below:
+
+.. code-block:: xml
+
+  <plugin>
+    <groupId>com.github.spotbugs</groupId>
+    <artifactId>spotbugs-maven-plugin</artifactId>
+    <version>3.0.6</version>
```

Comment:
Bump version to 3.1.0-SNAPSHOT

Reply:
Using SNAPSHOT is little bit trickly, I'll use next RC version.

Reply:
ok - if you keep with that version, the group id needs to be different.  I released that one under my groupId.

I even had released RC4.

```
<dependency>
    <groupId>com.github.hazendaz.spotbugs</groupId>
    <artifactId>spotbugs-maven-plugin</artifactId>
    <version>3.1.0-RC4</version>
</dependency>
```

The next releases of course will see removal of 'hazendaz'.

Reply:
Is there still something to do here, or can we merge this PR now?

Reply:
The usage should indicate the release I put out at the moment if this is merged now with it being fixed afterwards.  I've been hard at work getting lots of changes back into the original plugin.  In the interum some defects got created on the original unrelated to my work but for better good so waiting on a fix I sent to clear one of those.  However, that is build related so nothing really stops us now.  I need to do some final verifications and will cut a RC rather than final against RC5.  I understand RC6 was a bit buggie so don't know if that is wanted or not.  Thoughts?  I"ll be on tonight until this is done with something on official branding.

Reply:
@iloveeclipse I'm going to release RC5 just as soon as final integration tests finish.  I made some additional rebranding and think I got mostly everything aside from packaging.  I'm expecting a complete successful run of this and just waiting to hit go.  I'll start with RC5, get some feedback if that is ok then will do RC6 later this weekend and if all still ok I say lets let it rip with a solid 3.1.0 release.

Reply:
3.1.0.RC5 has been released.  It should be live in a couple of hours in maven central.

Reply:
Please bump this to show 3.1.0-RC6 then go ahead and merge it.